frontend/src/components/pages/profile/Profile.tsx

```diff
@@ -40,24 +40,19 @@ const Profile: React.FC<IParamsProps> = props => {
 
   return (
     <>
-      {/* TIME DECISION */}
-      {!userDetails ? (
-        <Redirect to="/login" />
-      ) : (
-        <Layout pageTitle={`${userDetails._id === _id ? 'My' : ''} Profile`}>
-          <div className="profile">
-            <div className="leftside">
-              <UserCard mainInfo={userDetails} boxShadow={2} isProfile />
-              <About about={userDetails.about} />
-              <FriendsList id={userDetails._id} friends={friends} />
-            </div>
-            <div className="rightside">
-              <AddPostForm _id={userDetails._id} />
-              <Posts posts={userPosts} />
-            </div>
+      <Layout pageTitle={`${userDetails._id === myInfo?._id ? 'My' : ''} Profile`}>
```

Comment:
better to create `isCurrentUser` method in `sessionStorageService `, so it can be reused in multiple places